lib/jekyll-import/importers/mt.rb

```diff
@@ -58,15 +65,24 @@ def self.specify_options(c)
       def self.process(options)
         options  = default_options.merge(options)
 
+        engine   = options.fetch('engine')
         dbname   = options.fetch('dbname')
         user     = options.fetch('user')
         pass     = options.fetch('password', "")
         host     = options.fetch('host', "localhost")
+        port     = options.fetch('port')
```

Comment:
Thanks for your patience, I'm new to Ruby. 

I've confirmed you were right about my use of `fetch`, and have updated the code to just use `options['port']` as you originally suggested, as that appears to be the best fit for this optional value.